app/aimodels/bertopic/routers/documents.py

```diff
@@ -0,0 +1,38 @@
+import os
+from typing import Union
+from fastapi import Depends, APIRouter
+from pydantic import UUID4
+from ..schemas.document import Document, DocumentCreate
+from app.dependencies import get_db
+from sqlalchemy.orm import Session
+from .. import crud
+from ..models.document import DocumentModel
+from app.core.errors import HTTPValidationError, ValidationError
+
+router = APIRouter(
+    prefix="/documents"
+)
+
+@router.post(
```

Comment:
issue entering document time, entered https://github.com/MIT-AI-Accelerator/c3po-model-server/issues/7 to address later

Reply:
agreed